app/views/admin/attachments/_form_legacy.html.erb

```diff
@@ -8,7 +8,7 @@
   <% end %>
 
   <% if attachable.allows_attachment_references? %>
-    <%= render 'reference_fields_legacy', attachable: attachable, form: form, attachment: attachment %>
+    <%= render 'reference_fields_legacy', attachable: attachable, form: form, attachment: attachment, root: nil %>
```

Comment:
It might be easier to pass in a heading size instead of root? i.e. 

```
<%= render 'reference_fields_legacy', attachable: attachable, form: form, attachment: attachment, heading_size: "m" %>
```

Reply:
Cheers both, thought that was looking a bit clunky. I've updated it in all the files that need it; think it looks a bit more elegant now.